src/components/Explore/Filters.tsx

```diff
@@ -66,39 +91,55 @@ const Filters = () => {
 
             <Stack direction="column" alignItems="center" spacing={1}>
                 <Box my={1} justifyContent="center">
-                    <DatePicker
-                        label="Start Date"
-                        value={startDate}
-                        renderInput={(params) => <TextField {...params} />}
-                        minDate={minDate}
-                        maxDate={maxDate}
-                        openTo="year"
-                        clearable
-                        onChange={(newVal) => {
-                            setStartDate(newVal);
-                        }}
-                    />
+                    <Stack direction="row">
+                        <DatePicker
+                            label="Start Date"
+                            value={startDate}
+                            renderInput={(params) => <TextField {...params} />}
+                            minDate={minDate}
+                            maxDate={maxStartDate}
+                            openTo="year"
+                            clearable
+                            onChange={(newVal) => {
+                                setStartDate(newVal);
+                            }}
+                        />
+                        <Box my={1}>
+                            <Button
+                                startIcon={<Icon baseClassName="icons">download</Icon>}
```

Comment:
Looks like `download` snuck into this! This should be `close` icon! :)

Reply:
oops

Reply:
Also the problem here is that the close button becomes a bit weird. Will need to make the button round so as to make it look good.